librz/core/cmd/cmd_open.c

```diff
@@ -1625,3 +1562,41 @@ RZ_IPI int rz_cmd_open(void *data, const char *input) {
 	}
 	return 0;
 }
+
+RZ_IPI RzCmdStatus rz_open_close_handler(RzCore *core, int argc, const char **argv) {
+	if (!rz_num_is_valid_input(NULL, argv[1])) {
+		RZ_LOG_ERROR("Invalid fd: %s\n", argv[1]);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	int fd = (int)rz_num_math(NULL, argv[1]);
+	if (!rz_core_file_close_fd(core, fd)) {
+		RZ_LOG_ERROR("Unable to find file descriptor %d\n", fd);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_open_close_all_handler(RzCore *core, int argc, const char **argv) {
+	rz_core_file_close_fd(core, -1);
+	rz_io_close_all(core->io);
+	rz_bin_file_delete_all(core->bin);
+
+	// TODO: Move to a-- ?
+	rz_analysis_purge(core->analysis);
+	// TODO: Move to f-- ?
+	rz_flag_unset_all(core->flags);
+	RZ_LOG_INFO("Close all files\n");
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_open_plugins_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
```

Comment:
Isn't this a duplicate of the `L` commands?

Reply:
Good point, yes i think so. I will review this